### ISSUE TITLE
Deployed Github pages URL has an extra column

### ISSUE THREAD

**Issue by tquiroga** [storybookjs/storybook-deployer#64]
Minor issue spotted on the output of the command when deploying to github pages:
```
=> Storybook deployed to: https://:myusername.github.io/my-repo/
✨  Done in 53.60s.
```

There is an extra column https:// **:** myusername[...].
That crashes the convenient Ctrl+click in terminal window to open a URL in browser.

**Comment by jvanaert** [comment]
any news on merging this ?

**Comment by tehpsalmist** [comment]
Is this issue being tracked? tested? Seems like an easy fix, and it's hard to believe that I'm the only dev out there using ssh for my git config...

**Comment by hipstersmoothie** [comment]
:rocket: Issue was released in [`v2.8.4`](https://github.com/storybookjs/storybook-deployer/releases/tag/v2.8.4) :rocket:

**Comment by hipstersmoothie** [comment]
:rocket: Issue was released in [`v2.8.4`](https://github.com/storybookjs/storybook-deployer/releases/tag/v2.8.4) :rocket:
